tests/articulation_tests.ts

```diff
@@ -250,53 +250,59 @@ function verticalPlacement2(options: TestOptions, contextBuilder: ContextBuilder
+      .addModifier(new Articulation(0xe4c5 /*fermataShortBelow*/), 0)
+      .addModifier(new Articulation(0xe1e7 /*augmentationDot*/).setPosition(ModifierPosition.BELOW), 0)
+      .addModifier(new Articulation(0xe4a5 /*articTenutoBelow*/), 0),
     staveNote({ keys: ['c/5'], duration: 'q' })
-      .addModifier(new Articulation('fermataLongBelow'), 0)
-      .addModifier(new Articulation('augmentationDot').setPosition(ModifierPosition.BELOW), 0)
-      .addModifier(new Articulation('articTenutoBelow'), 0),
+      .addModifier(new Articulation(0xe4c7 /*fermataLongBelow*/), 0)
+      .addModifier(new Articulation(0xe1e7 /*augmentationDot*/).setPosition(ModifierPosition.BELOW), 0)
+      .addModifier(new Articulation(0xe4a5 /*articTenutoBelow*/), 0),
     staveNote({ keys: ['f/4'], duration: 'q' })
-      .addModifier(new Articulation('augmentationDot').setPosition(ModifierPosition.BELOW), 0)
-      .addModifier(new Articulation('articTenutoBelow'), 0)
-      .addModifier(new Articulation('fermataVeryShortBelow'), 0),
+      .addModifier(new Articulation(0xe1e7 /*augmentationDot*/).setPosition(ModifierPosition.BELOW), 0)
+      .addModifier(new Articulation(0xe4a5 /*articTenutoBelow*/), 0)
+      .addModifier(new Articulation(0xe4c3 /*fermataVeryShortBelow*/), 0),
     staveNote({ keys: ['g/4'], duration: 'q', stemDirection: Stem.DOWN })
-      .addModifier(new Articulation('augmentationDot').setPosition(ModifierPosition.BELOW), 0)
-      .addModifier(new Articulation('articTenutoBelow'), 0)
-      .addModifier(new Articulation('fermataVeryLongBelow'), 0),
+      .addModifier(new Articulation(0xe1e7 /*augmentationDot*/).setPosition(ModifierPosition.BELOW), 0)
+      .addModifier(new Articulation(0xe4a5 /*articTenutoBelow*/), 0)
+      .addModifier(new Articulation(0xe4c9 /*fermataVeryLongBelow*/), 0),
     staveNote({ keys: ['c/5'], duration: 'q' })
-      .addModifier(new Articulation('augmentationDot').setPosition(ModifierPosition.BELOW).setBetweenLines(), 0)
-      .addModifier(new Articulation('articTenutoBelow').setBetweenLines(), 0)
-      .addModifier(new Articulation('fermataBelow'), 0),
+      .addModifier(
+        new Articulation(0xe1e7 /*augmentationDot*/).setPosition(ModifierPosition.BELOW).setBetweenLines(),
+        0
+      )
+      .addModifier(new Articulation(0xe4a5 /*articTenutoBelow*/).setBetweenLines(), 0)
+      .addModifier(new Articulation(0xe4c1 /*fermataBelow*/), 0),
     staveNote({ keys: ['a/5'], duration: 'q', stemDirection: Stem.DOWN })
-      .addModifier(new Articulation('fermataAbove'), 0)
-      .addModifier(new Articulation('augmentationDot').setPosition(ModifierPosition.ABOVE), 0)
-      .addModifier(new Articulation('articTenutoAbove'), 0),
+      .addModifier(new Articulation(0xe4c0 /*fermataAbove*/), 0)
+      .addModifier(new Articulation(0xe1e7 /*augmentationDot*/).setPosition(ModifierPosition.ABOVE), 0)
+      .addModifier(new Articulation(0xe4a4 /*articTenutoAbove*/), 0),
```

Comment:
Is there a benefit to using numeric codepoints? Later on we have to translate it via String.fromCodePoint() or String.fromCharCode().

Should we refactor this class to accept a string, and then require the developer to pass in the correct SMuFL glyph directly?

```
String.fromCharCode(0xE4C0) === '\uE4C0'
```

So instead of using numbers, we could use the `\uXXXX` escape code for unicode characters.

Reply:
done